lib/karafka/setup/config.rb

```diff
@@ -89,6 +89,9 @@ class Config
         #   are stored for further processing. Note, that each item in the queue represents a
         #   response from a single broker
         setting :fetcher_max_queue_size, 10
+        # option assignment_strategy [Object] a strategy determining the assignment of
+        #   partitions to the consumers.
+        setting :assignment_strategy, Kafka::RoundRobinAssignmentStrategy.new
```

Comment:
could we wrap it with Karafka simpledelegator? Lets say in a `Karafka::AssignmentStrategies::RundRobin`? I would prefer not to directly show the Kafka internals if possible

Reply:
Sure